packages/bpk-scrim-utils/src/BpkScrim.js

```diff
@@ -16,21 +16,38 @@
  * limitations under the License.
  */
 
+import PropTypes from 'prop-types';
 import React from 'react';
 import { TransitionInitialMount, cssModules } from 'bpk-react-utils';
 
 import STYLES from './bpk-scrim.scss';
 
 const getClassName = cssModules(STYLES);
 
-const BpkScrim = () => (
-  <TransitionInitialMount
-    appearClassName={getClassName('bpk-scrim--appear')}
-    appearActiveClassName={getClassName('bpk-scrim--appear-active')}
-    transitionTimeout={200}
-  >
-    <div className={getClassName('bpk-scrim')} />
-  </TransitionInitialMount>
-);
+const BpkScrim = (props) => {
+  const classNames = [getClassName('bpk-scrim')];
+
+  if (props.visibleBackground) {
```

Comment:
It looks like the behaviour of the `visibleBackground` is to prevent the scrim from removing opacity at mobile breakpoints. If this is the case then I'd prefer to flip the logic and make the lack of opacity on mobile the special case i.e. `hideBackpackOnMobile` or something. This would mean a breaking change for the `bpk-scrim-utils` packages - but the main consumption of this behaviour is encapsulated within modals and drawers therefore it shouldnt be too much (if any) disruption for consumers.